codegen/src/main/twirl/templates/PlayScalaServer/Router.scala.txt

```diff
@@ -0,0 +1,37 @@
+@*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ *@
+
+@(service: akka.grpc.gen.scaladsl.Service)
+
+package @service.packageName
```

Comment:
What's the default packageName, out of interest?

Reply:
It's determined as described in https://scalapb.github.io/customizations.html#package-options . By default we enable the `flat_package` option.